src/rpc/mod.rs

```diff
@@ -188,29 +184,6 @@ where
     module.register_async_method(AUTH_VERIFY, auth_verify::<DB>)?;
     // Beacon API
     module.register_async_method(BEACON_GET_ENTRY, beacon_get_entry::<DB>)?;
-    // Chain API
-    module.register_async_method(CHAIN_GET_MESSAGE, chain_get_message::<DB>)?;
```

Comment:
Should access control code be simplified as well?
```rust
// Chain API
    access.insert(chain_api::CHAIN_GET_MESSAGE, Access::Read);
    access.insert(chain_api::CHAIN_EXPORT, Access::Read);
    access.insert(chain_api::CHAIN_READ_OBJ, Access::Read);
...
```

Reply:
(it would require swapping out middleware etc, which I'm all in favor of)